src/test/java/core/basesyntax/service/impl/ReaderFromFileImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.service.Reader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+class ReaderFromFileImplTest {
+    private static final String VALID_INPUT_FILE = "src/main/resources/inputData.csv";
```

Comment:
We should not use main resources in the tests, create a duplicate to separate them. 
Exception can be a usage of SQL scripts in main to set up the test DB but we will talk about that later on